werkzeug/exceptions.py

```diff
@@ -672,7 +672,27 @@ def __call__(self, code, *args, **kwargs):
             raise LookupError('no exception for %r' % code)
         raise self.mapping[code](*args, **kwargs)
 
-abort = Aborter()
+
+def abort(status, *args, **kwargs):
```

Comment:
Good, I shall change my project and open a PR to fix Aborter in master? or in the maintenance branch?